docker/openemr/6.1.0/Dockerfile

```diff
@@ -54,9 +54,9 @@ ENV APACHE_LOG_DIR=/var/log/apache2
 COPY php.ini /etc/php8/php.ini
 COPY openemr.conf /etc/apache2/conf.d/
 #add runner and auto_configure and prevent auto_configure from being run w/o being enabled
-COPY run_openemr.sh autoconfig.sh auto_configure.php /var/www/localhost/htdocs/openemr/
+COPY openemr.sh ssl.sh xdebug.sh auto_configure.php /var/www/localhost/htdocs/openemr/
```

Comment:
wouldn't place the ssl.sh and xdebug.sh scripts in the openemr directory (then can get to the via web browser; prob impossible to run, but wouldn't risk it; especially the xdebug script could be a nasty security vulnerability if bad actor figured out how to run it)

Reply:
The ssl and xdebug scripts are no more vulnerable than the openemr script itself -- if /that's/ a problem we should fix it, and if it's not we can leave it alone, right?

Reply:
You have a point. But the openemr script is protected by settings (for example, xdebug will not be run unless that setting is on, where is somebody can run xdebug.sh directly then that protection won't work)

Reply:
One of the reasons i didn't include xdebug in the production line was because i honestly didn't no how secure i could keep it, though.

Reply:
I mean, it should be impossible to run a bash script from a browser. I think if you simply put following in the xdebug.sh script, then I can sleep better :) :
```
if [ "$XDEBUG_IDE_KEY" != "" ] ||
   [ "$XDEBUG_ON" == 1 ]; then
```